chaincode/compute_plan.go

```diff
@@ -31,7 +30,7 @@ func (inpTraintuple *inputTraintuple) Fill(inpCP inputComputePlanTraintuple, tra
 	for _, InModelID := range inpCP.InModelsIDs {
 		inModelKey, ok := traintupleKeysByID[InModelID]
 		if !ok {
-			return fmt.Errorf("model ID %s not found", InModelID)
+			return errors.BadRequest("model ID %s not found", InModelID)
```

Comment:
shouldn't this be a NotFound error?

Reply:
The ID are found in the input, so if it's missing it's due to the arguments so a BadRequest was more logical what do you think?

Reply:
indeed